pkg/ccl/backupccl/backup_planning.go

```diff
@@ -701,14 +702,17 @@ func backupPlanHook(
 				initialDetails.Destination.Subdir = latestFileName
 			} else if subdir != "" {
 				initialDetails.Destination.Subdir = "/" + strings.TrimPrefix(subdir, "/")
-				// Deprecation notice for `BACKUP INTO` syntax with an explicit subdir.
-				// Remove this once the syntax is deleted in 22.2.
-				p.BufferClientNotice(ctx,
-					pgnotice.Newf("BACKUP commands with an explicitly specified"+
-						" subdirectory will be removed in a future release. Users can create a full backup via `BACKUP ... "+
-						"INTO <collectionURI>`, or an incremental backup on the latest full backup in their "+
-						"collection via `BACKUP ... INTO LATEST IN <collectionURI>`"))
-
+				if _, err := time.Parse(DateBasedIntoFolderName,
```

Comment:
Won't it be more intuitive to check for a previous backup in the resolved destination instead of checking the nature of the subdir path passed in by the user?

My understanding is that we only want `subdir` to be equal to something we put in place when running a base `BACKUP INTO <foo>` and so it should be sufficient to check if a BACKUP_MANIFEST exists in the resolved path?

Reply:
will respond in  #79799!